packages/azure-themes/src/stories/components/messageBar.tsx

```diff
@@ -0,0 +1,200 @@
+    isMultiline={false}
+    onDismiss={p.resetChoice}
+    dismissButtonAriaLabel="Close"
+    truncated={true}
+    overflowButtonAriaLabel="See more"
+  >
+    <b>Blocked MessageBar - single line, with dismiss button and truncated text.</b> Truncation is not available if you
+    use action buttons or multiline and should be used sparingly. Lorem ipsum dolor sit amet, consectetur adipiscing
+    condimentum mauris.
+  </MessageBar>
+);
+
+const SevereExample = (p: IExampleProps) => (
+  <MessageBar
+    messageBarType={MessageBarType.severeWarning}
+    actions={
+      <div>
+        <MessageBarButton>Yes</MessageBarButton>
+        <MessageBarButton>No</MessageBarButton>
+      </div>
+    }
+  >
+    SevereWarning MessageBar with action buttons which defaults to multiline.
+    <Link href="www.bing.com" target="_blank">
+      Visit our website.
+    </Link>
+  </MessageBar>
+);
+
+const SuccessExample = () => (
+  <MessageBar
+    actions={
+      <div>
+        <MessageBarButton>Yes</MessageBarButton>
+        <MessageBarButton>No</MessageBarButton>
+      </div>
+    }
+    messageBarType={MessageBarType.success}
+    isMultiline={false}
+  >
+    Success MessageBar with single line and action buttons.
+    <Link href="www.bing.com" target="_blank">
+      Visit our website.
+    </Link>
+  </MessageBar>
+);
+
+const WarningExample = (p: IExampleProps) => (
+  <MessageBar
+    messageBarType={MessageBarType.warning}
+    isMultiline={false}
+    onDismiss={p.resetChoice}
+    dismissButtonAriaLabel="Close"
+    actions={
+      <div>
+        {/* <MessageBarButton>Action</MessageBarButton> */}
+        {/* <> */}
+        <DefaultButton>Action</DefaultButton>
+        <PrimaryButton>Action</PrimaryButton>
+        {/* </> */}
+      </div>
+    }
+  >
+    Warning MessageBar content.
+    <Link href="www.bing.com" target="_blank">
+      Visit our website.
+    </Link>
+  </MessageBar>
+);
+
+const choiceOptions = [
+  {
+    key: 'default',
+    text: 'Default',
+  },
+  {
+    key: 'error',
+    text: 'Error MessageBar',
+  },
+  {
+    key: 'blocked',
+    text: 'Blocked MessageBar',
+  },
+  {
+    key: 'severe',
+    text: 'SevereWarning MessageBar',
+  },
+  {
+    key: 'success',
+    text: 'Success MessageBar',
+  },
+  {
+    key: 'warning',
+    text: 'Warning MessageBar - single line',
+  },
+  {
+    key: 'warning2',
+    text: 'Warning MessageBar - multiline',
+  },
+  {
+    key: 'all',
+    text: 'Show All',
+  },
+];
+
+export const MessageBarBasicExample: React.FunctionComponent = () => {
+  const [choice, setChoice] = React.useState<string | undefined>(undefined);
+  const showAll = choice === 'all';
+
+  const resetChoice = React.useCallback(() => setChoice(undefined), []);
+
+  return (
+    <Stack {...horizontalStackProps}>
+      <StackItem disableShrink>
+        <ChoiceGroup
+          styles={choiceGroupStyles}
+          label="Select a MessageBar Example Below. To test in narrator, show one message at a time."
```

Comment:
Very helpful description, and good call on adding some functionality to test the narrator! 💯